app/infrastructure/broadcaster.py

```
@@ -1,44 +1,41 @@
-                return
-            subscribers = list(self._subscribers)
-
-        for queue in subscribers:
-            with contextlib.suppress(asyncio.QueueFull):
-                queue.put_nowait(metric)
-
-    async def shutdown(self) -> None:
-        async with self._lock:
-            for queue in list(self._subscribers):
-                while True:
+    def __init__(self, channel: Optional[str] = "metrics"):
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)